docs/components/DrawerDocs.js

```diff
@@ -140,17 +150,72 @@ const DrawerDocs = React.createClass({
         <p>Default: ''</p>
         <p>This will be displayed in the header of the drawer component.</p>
 
+        <h5>DEPRECATED: navConfig <label>Object</label></h5>
+        <p>This object requires 3 properties:</p>
+        <ul style={styles.unorderdLists}>
+          <li style={styles.listItem}><h5 style={styles.h5ListItem}>label <label>String</label></h5>This will be displayed between the two arrow buttons.</li>
+          <li style={styles.listItem}><h5 style={styles.h5ListItem}>onPreviousClick <label>Function</label></h5> This function will be called when the left arrow is clicked.</li>
+          <li style={styles.listItem}><h5 style={styles.h5ListItem}>onNextClick <label>Function</label></h5> this function will be called when the right arrow is clicked.</li>
+        </ul>
+
         <h3>Example</h3>
         <Markdown>
   {`
+
+    _handleDrawerClose () {
+      this.setState({
+        demoDrawerOpen: false
+      });
+    },
+
     <Drawer
-      buttonPrimaryColor='#333333'
-      onClose={this._handleCloseDrawerClick}
-      showScrim={false}
+      breakPoints={{ large: 1200, medium: 1100 }}
+      contentStyle={styles.content}
+      headerMenu={(
+        <HeaderMenu
+          handleButtonClick={this._handleSimpleSelectClick}
+          handleScrimClick={this._handleSimpleSelectClick}
+          showSimpleSelectMenu={this.state.showMenu}
+        />
+      )}
+      onClose={this._handleDrawerClose}
       title='Demo Drawer'
     >
-      //Content here
+      // Content Here
+
     </Drawer>
+
+    // Component To Pass As A Prop
+    function HeaderMenu ({ handleButtonClick, handleScrimClick, showSimpleSelectMenu = false }) {
```

Comment:
I also think we should add the header menu as a separate component in `src/components/HeaderMenu.js`